src/include/fann_data.h

```diff
@@ -491,6 +491,7 @@ struct fann
 
 	/* The learning momentum used for backpropagation algorithm. */
 	float learning_momentum;
+	float learning_l2_norm;
```

Comment:
Is `learning_l2_norm` the best name for this parameter? It's actually not the L2 norm, but rather the weight of the L2 norm relative to the error function.